### DISCUSSION TITLE
Model has NULL residuals

### DISCUSSION THREAD

**Discussion by galuhchynintya** [teamalgoritma/community#907] [Q&A] [answered]
Dear tim algoritma,
Saya sedang melakukan uji asumsi pada model time series. Tapi terdapat error --> Error in ts(x) : 'ts' object must have one or more observations
Tapi hal ini terjadi pada model Holt-Winters dan TBATS, pada model SARIMA tidak.
-Model Holt-Winters
![image](https://github.com/user-attachments/assets/77e42cda-606e-426d-8c72-ee022ec19d9f)
-Model SARIMA
![image](https://github.com/user-attachments/assets/674a1a7c-edd0-4920-93cc-f8a9e04038da)
-Model TBATS
![image](https://github.com/user-attachments/assets/cac2ddb3-150b-4e86-b853-22696565e6a6)
Setelah saya cek pada model$residuals, tidak terdapat residual pada model Holt-Winters dan TBATS
![image](https://github.com/user-attachments/assets/e5039669-97bf-43e6-8015-23e039934437)
Mohon pencerahannya, apakah model bisa tidak memiliki nilai residual atau ada proses yang kemungkinan salah?
Terima kasih

**Comment by adizaen** [comment]
Dear Bu @galuhchynintya 

Terima kasih atas pertanyaannya Bu. Untuk saat ini kami masih mencoba untuk menganalisis error yang terjadi Bu. Jika sudah ada jawaban, akan segera kami informasikan di sini ya Bu. Jika berkenan, apakah Bu Galuh bisa mengirimkan file .rmd dan datasetnya Bu? Agar kami bisa cek terlebih dahulu terkait error yang terjadi.

Terima kasih Bu Galuh.

**Comment by galuhchynintya** [comment]
Dear mas Adi,
Untuk data set nya saya menggunakan data dari tsdl https://pkg.yangzhuoranyang.com/tsdl/. Saya juga coba datanya dilakukan preprocessing dengan sqrt() dan scale() tapi masih null residualnya.
Berikut saya lampirkan file rmd nya. Terima kasih
[LBB_TS_1.zip](https://github.com/user-attachments/files/16689829/LBB_TS_1.zip)

**Comment by adizaen** [comment, marked as answer]
Dear Bu @galuhchynintya

Setelah kami telaah lebih lanjut, bisa kami informasikan bahwa untuk uji asumsi hanya dibutuhkan pada ARIMA atau SARIMA Bu. Sedangkan untuk model Holt-Winters dan TBATS tidak membutuhkan uji asumsi dan memang tidak tersedia uji asumsi untuk kedua model tersebut Bu. Mungkin seperti itu jawaban yang dapat kami berikan Bu,

Terima kasih Bu Galuh.

**Reply by galuhchynintya** [reply]
Dear mas Adi,
Ok noted mas Adi, terima kasih atas insight nya.
